src/zoid/buttons/component.jsx

```diff
@@ -20,7 +20,7 @@ import { isFundingEligible } from '../../funding';
 import { containerTemplate } from './container';
 import { PrerenderedButtons } from './prerender';
 import { applePaySession, determineFlow, isSupportedNativeBrowser, createVenmoExperiment,
-    getVenmoExperiment, createNoPaylaterExperiment, getNoPaylaterExperiment, getRenderedButtons } from './util';
+    createNoPaylaterExperiment, getRenderedButtons, getButtonsExperiments } from './util';
```

Comment:
Can we change to `getButtonExperiments`?

Reply:
sure!